packages/wasm-terminal/src/process/process.ts

```diff
@@ -114,16 +114,23 @@ export default class Process {
       }
 
       let error = "Unknown Error";
+      let isUserError = e.user !== undefined;
 
       if (e.code !== undefined) {
         error = `exited with code: ${e.code}`;
       } else if (e.signal !== undefined) {
         error = `killed with signal: ${e.signal}`;
-      } else if (e.user !== undefined) {
+      } else if (isUserError) {
         error = e.message;
```

Comment:
You can do directly here (removing new line 117 and changes after line 126)

```js
      } else if (e.user !== undefined) {
        end();
      }
      console.error(e);
      this.errorCallback(error, this.wasmFs.toJSON());
```

It makes the changes much easier to read (and less changes)

Reply:
Cool Sounds good :)